modules/ocf/manifests/init.pp

```diff
@@ -12,6 +12,7 @@
   include ocf::munin::node
   include ocf::networking
   include ocf::packages
+  include ocf::prometheus_node
```

Comment:
I'd organize this like the munin class above, e.g. `ocf::prometheus::node`, to maintain the existing pattern

Reply:
Does it make sense to have a folder with just one file in it? That's the only reason why I did it this way.